docs/source/api/v4/servers.rst

```diff
@@ -236,6 +237,7 @@ Allows a user to create a new server.
 
 :Auth. Required: Yes
 :Roles Required: "admin" or "operations"
+:Permissions Required: SERVER:CREATE, CACHE-GROUP:UPDATE, DELIVERY-SERVICE:UPDATE, CDN:UPDATE, SERVER:READ, DELIVERY-SERVICE:READ, CDN:READ, PHYSICAL-LOCATION:READ, CACHE-GROUP:READ, TYPE:READ, PROFILE:READ
```

Comment:
Does this need `DELIVERY-SERVICE:UPDATE`, `CACHE-GROUP:UPDATE`, and `CDN-UPDATE`?